wear/src/main/AndroidManifest.xml

```diff
@@ -43,6 +43,24 @@
 
         <!-- To show confirmations and failures -->
         <activity android:name="androidx.wear.activity.ConfirmationActivity" />
+
+        <!-- Tiles -->
+        <service
+            android:name=".tiles.FavoriteEntitiesTile"
+            android:label="@string/favorite_entities"
+            android:description="@string/favorite_entities"
```

Comment:
Maybe add some kind of different description like "Select up to 7 entities" because in the Wear OS app I see the name repeating :)

![image](https://user-images.githubusercontent.com/1634145/141025165-7c54f4f1-ae47-4e51-9073-aaf144b92eed.png)

Reply:
Good suggestion!

I also changed the name of the tile to Shortcuts. Also because it is really separate from the favorites we have in the app.